etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/results/SendResultUseCase.java

```diff
@@ -17,6 +20,13 @@ public static SendResultUseCase getInstance() {
 
     @Override
     public void convertAndSend(Result<?> result) throws UnableToSendMessageException {
-        sender.send(result);
+
```

Comment:
I know we're not worrying about metadata for results quite yet but it looks fine to me. We can leave it commented for now until we understand the whole criteria for results metadata.

Reply:
This method is implemented in #859